dowhy/gcm/independence_test/kernel.py

```diff
@@ -2,59 +2,54 @@
 future.
 """
 
-from typing import Callable, List, Union, Optional, Tuple
+from typing import Callable, List, Union, Optional
 
 import numpy as np
 import scipy
+from causallearn.utils.KCI.KCI import KCI_CInd, KCI_UInd
```

Comment:
The causal-learn package is currently licensed as GPL.  Since our library is MIT licensed, it will be difficult for us to have code integration like this unless/until the license is changed.